QUICK_START.md

```diff
@@ -80,6 +81,9 @@ $my_mailbox = Get-PolarisO365Mailbox -Token $token -PolarisURL $url -Subscriptio
 # get our onedrive user
 $my_onedrive = Get-PolarisO365OneDrive -Token $token -PolarisURL $url -SubscriptionId $my_sub.id -SearchString 'arif'
 
+# get our SharePoint sites

```

Comment:
Can we also add instruction for getting document library only or getting both items?